client/landing/stepper/declarative-flow/internals/index.tsx

```diff
@@ -136,7 +136,11 @@ export const FlowRenderer: React.FC< { flow: Flow } > = ( { flow } ) => {
 		setPreviousProgress( stepProgress?.progress ?? 0 );
 	};
 
-	const stepNavigation = flow.useStepNavigation( currentStepRoute, _navigate );
+	const stepNavigation = flow.useStepNavigation(
+		currentStepRoute,
+		_navigate,
+		flowSteps.map( ( step ) => step.slug )
```

Comment:
I added the `steps` argument here.

It seems [when it was created](https://github.com/Automattic/wp-calypso/commit/b57297fac00a56cf103f3235e33f6636ddca1f49), the [type already had the steps argument](https://github.com/Automattic/wp-calypso/commit/b57297fac00a56cf103f3235e33f6636ddca1f49#diff-b07941382fbff706bdda39e9b551a1827566a6d90d9ed3f6e286ad7a9e62d02dR25), but it was never used.